Converters/Tekla/Speckle.Converter.Tekla2024/ToSpeckle/Helpers/DisplayValueExtractor.cs

```diff
@@ -38,13 +39,21 @@ public IEnumerable<Base> GetDisplayValue(TSM.ModelObject modelObject)
         }
         break;
 
+      // logic to send reinforcement as solid
       case TSM.Reinforcement reinforcement:
         if (reinforcement.GetSolid() is TSM.Solid reinforcementSolid)
         {
           yield return _meshConverter.Convert(reinforcementSolid);
         }
         break;
 
+      case TSM.Grid grid:
+        foreach (var gridLine in _gridHandler.GetGridLines(grid))
```

Comment:
I would follow the same "converter" logic instead of "handler" to make it consistent